modules/cnf-about-topology-aware-lifecycle-operator-config.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-topology-aware-lifecycle-operator-config_{context}"]
+= About the Topology Aware Lifecycle Operator configuration
+
+The Topology Aware Lifecycle Operator manages the deployment of policies for one or more {product-title} clusters. The Operator allows you to control the following actions:
+* The timing of the update
+* The number of {rh-rhacm}-managed clusters
+* The update order of the clusters
+
+The Operator supports the orchestration of the following update scenarios:
```

Comment:
I think it would make more sense to use TALO instead of "The Operator" throughout this documentation.